srcpkgs/liburing/template

```diff
@@ -1,16 +1,16 @@
 # Template file for 'liburing'
 pkgname=liburing
-version=2.1
+version=2.2
 revision=1
 build_style=configure
 configure_args="--mandir=/usr/share/man"
 make_build_args="-C src"
 short_desc="Linux-native io_uring I/O access library"
-maintainer="Anthony Iliopoulos <ailiop@altatus.com>"
+maintainer="Dragon Friend <friend-dragon@proton.me>"
```

Comment:
@ailiop-git can they adopt this from you?

Reply:
sure, go ahead